python/dgl/convert.py

```diff
@@ -20,394 +21,197 @@
     'to_hetero',
     'to_homo',
     'from_scipy',
+    'bipartite_from_scipy',
     'from_networkx',
+    'bipartite_from_networkx',
     'to_networkx',
 ]
 
 def graph(data,
-          ntype='_N', etype='_E',
+          ntype=None, etype=None,
           num_nodes=None,
-          validate=True,
-          formats=['coo', 'csr', 'csc'],
```

Comment:
This argument is completely removed, right?

Reply:
Right. `validate` is always performed and `formats` is left to `DGLGraph.formats`.